src/python_testing/TC_CADMIN_1_9.py

```diff
@@ -105,7 +105,13 @@ async def CommissionAttempt(
             errcode = await self.CommissionOnNetwork(setupPinCode)
             logging.info('Commissioning complete done. Successful? {}, errorcode = {}'.format(errcode.is_success, errcode))
             asserts.assert_false(errcode.is_success, 'Commissioning complete did not error as expected')
-            asserts.assert_true(errcode.sdk_code == expectedErrCode, 'Unexpected error code returned from CommissioningComplete')
+            # TODO: Adding try or except clause here as the errcode code be either 50 for timeout or 3 for incorrect state at this time
+            # until issue mentioned in https://github.com/project-chip/connectedhomeip/issues/34383 can be resolved
+            try:
+                asserts.assert_true(errcode.sdk_code == expectedErrCode,
+                                    'Unexpected error code returned from CommissioningComplete')
+            except Exception:
```

Comment:
except Exception is quite broad. Can you change this to just
asserts.assert_in(errcode.sdk_code, [expectedErrCode, 3], 'Unexpected error code returned from CommissioningComplete')